packages/fxa-settings/src/lib/hooks/useTotpSetup/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { createContext, useContext } from 'react';
+import { useAsync } from 'react-async-hook';
```

Comment:
Hmm, this is only the 2nd place now where this is used. Is it a good candidate for dep removal? Not sure off the top of my head what it's providing for us.

Reply:
I swear it was, but I must have fumbled something. I'll repush with both this import and context removed.